src/threading/SerialTypes.cc

```diff
@@ -561,6 +561,9 @@ Val* Value::ValueToVal(const std::string& source, const Value* val, bool& have_e
 			for ( int j = 0; j < val->val.set_val.size; j++ )
 				{
 				Val* assignval = ValueToVal(source, val->val.set_val.vals[j], have_error);
+				if ( have_error )
+					return nullptr;
```

Comment:
This can leak the `t = new TableVal` allocated just above this loop.  Should be able to simply make `t` an `IntrusivePtr` instead to avoid that.